codespell_lib/data/dictionary.txt

```diff
@@ -9770,7 +9773,7 @@ dordered->ordered
 dorment->dormant
 dorp->drop
 dosen't->doesn't
-dosen->doesn
```

Comment:
Can you add this typo too, although I suspect this was an attempt to do something clever with apostrophes as per an issue I now can't find.

Reply:
It was added in https://github.com/codespell-project/codespell/pull/1330.

Reply:
Thanks, sorry I meant I couldn't find the issue about corrections which never match the default regex.